lib/test/tasks/run_file_size_checks.rb

```diff
@@ -11,9 +11,10 @@ class Test::Tasks::RunFileSizeChecks < Pallets::Task
     'check_ins*.js' => (183..193),
     'check_ins_index*.js' => (5..15),
     'copy_to_clipboard*.js' => (2..12),
-    'emoji_picker*.js' => (235..245),
-    'groceries*.css' => (93..103),
-    'groceries*.js' => (410..420),
+    'emoji_picker*.css' => (8..18),
+    'emoji_picker*.js' => (250..260),
+    'groceries*.css' => (108..118),
+    'groceries*.js' => (429..439),
```

Comment:
It's a bit disappointing to be adding this page weight. To some extent, it's the result of the fact that we now have two toaster libraries, instead of just one. (We are adding `vue-toastification` in addition to `toastify-js`.) The advantage of `toastify-js` is that it can be used independently of Vue, which we sometimes need, but the advantage of `vue-toastification` is that it can render Vue components (in addition to simple strings), which is really helpful when trying to do something relatively complex in the toast, such as seen in the `DeletedItemToast.vue` component added in this PR. So, to some extent, I do think it makes sense to have both, for now. Also, I think that https://davidrunger.atlassian.net/browse/DEV-100 will make it so that, in these apps, we won't have double toaster libraries, anymore, so I think that these page weight might go back down when that ticket is implemented.